# HG changeset patch
# User David Meyer <pdox@google.com>
# Date 1302185330 25200
# Branch pnacl-sfi
# Node ID 38c54bef28493f72374c5cb8c6f488991fd56c06
# Parent 8f2bc6b5df85b4cd8baf8755515519b190310492
Fix typo in ARM assembly libgcc macros

 From llvm-gcc-pnacl-0002-74-38c54bef28493f72374c5cb8c6f488991fd56c06.patch

diff -r 8f2bc6b5df85 gcc/config/arm/lib1funcs.asm
--- a/gcc/config/arm/lib1funcs.asm	Fri Mar 18 08:34:49 2011 -0700
+++ b/gcc/config/arm/lib1funcs.asm	Mon Jun 13 17:55:52 2011 -0700
@@ -213,28 +213,28 @@
 	bx	lr
 #else
 #define RETLDM \
-	ldr     lr, [sp], #8
-  sfi_bx  lr
+	ldr     lr, [sp], #8 ; \
+        sfi_bx  lr
 /* APPLE LOCAL begin v7 support. Merge from mainline */
 #if defined (__thumb2__)
 #define RETLDM1(...) \
-	pop   {__VA_ARGS__, lr}
-  sfi_bx lr
+	pop   {__VA_ARGS__, lr} ; \
+        sfi_bx lr
 #define RETLDM2(cond,...) \
-	pop##cond   {__VA_ARGS__, lr}
-  sfi_bx lr
+	pop##cond   {__VA_ARGS__, lr} ; \
+        sfi_bx lr
 #else
 #define RETLDM1(...) \
-	ldmia   sp!, {__VA_ARGS__, lr}
-  sfi_bx lr
+	ldmia   sp!, {__VA_ARGS__, lr} ; \
+        sfi_bx lr
 #define RETLDM2(cond,...) \
-	ldm##cond##ia   sp!, {__VA_ARGS__, lr}
-  sfi_bx lr
+	ldm##cond##ia   sp!, {__VA_ARGS__, lr} ; \
+        sfi_bx lr
 #endif
 /* APPLE LOCAL end v7 support. Merge from mainline */
 #define RETLDM_unwind(addr) \
-	ldr	lr, [sp], #8
-  sfi_bx lr
+	ldr	lr, [sp], #8 ; \
+        sfi_bx lr
 #endif
 
 /* APPLE LOCAL begin v7 support. Merge from mainline */
